ffi/value.cpp

```diff
@@ -404,11 +404,6 @@ LLVMPY_SetValueName(LLVMValueRef Val, const char *Name) {
 API_EXPORT(LLVMModuleRef)
 LLVMPY_GetGlobalParent(LLVMValueRef Val) { return LLVMGetGlobalParent(Val); }
 
-API_EXPORT(LLVMTypeRef)
```

Comment:
I believe this was introduced with #1051, and when I rebased my patch on top of it I thought it would make sense to keep `LLVMPY_GlobalGetValueType` in `type.cpp` next to other similar functions like `LLVMPY_GetElementType` and `LLVMPY_GetReturnType`. But it makes no odds to me, I'm happy to leave it in `value.cpp`.

Reply:
I've moved this back to `value.cpp`, I don't think we need to make this change in this patch.